libs/@local/harpc/tower/src/body/boxed.rs

```diff
@@ -0,0 +1,119 @@
+use core::{
+    fmt::{self, Debug, Formatter},
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use bytes::Buf;
+
+use super::{empty::Empty, Body, BodyState, Frame};
+
+pub struct BoxBody<D, C, E> {
```

Comment:
Could you add a short doc to the structs in this module? It's clear that it's wrapping some `dyn Box` but not why.

Reply:
see [28d6155](https://github.com/hashintel/hash/pull/4713/commits/28d6155d99f98bf93d7c75a8f44e880900a488c0)